### ISSUE TITLE
Some question about training！

### ISSUE THREAD

**Issue by s-bei** [ZHKKKe/MODNet#39]
Hello,  your work is excellent！
But, I want to know how your pre-trained model is trained,and I want to know how your loss function is represented in your code.
Can you publish more your code details about the training process?
Can you contact me through email?
an_shipeng@tju.edu.cn

**Comment by ZHKKKe** [comment]
Hi, thanks for your attention.

As said in our README file, our training code will be published next month. 
I am sorry that we cannot publish it now.

**Comment by s-bei** [comment]
Thank you very much for taking time out of your busy schedule to reply to me!

I am very interested in your work and look forward to your announcement!I will continue to follow your Github homepage!

thank you





On 12/25/2020 21:51，Zhanghan Ke<notifications@github.com> wrote：

Hi, thanks for your attention.

As said in our README file, our training code will be published next month.
I am sorry that we cannot publish it now.

**Comment by ZHKKKe** [comment]
Hi, our main training code is available now.
